iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -55,13 +55,9 @@
     </None>
   </ItemGroup>
 
-  <!--Project reference dependencies-->
-  <ItemGroup>
-    <ProjectReference Include="..\..\service\src\Microsoft.Azure.Devices.csproj" />
-  </ItemGroup>
-
   <!--Nuget package dependencies-->
   <ItemGroup>
+    <PackageReference Include="Azure.Core" Version="1.26.0" />
```

Comment:
It seems that we have dependency on some libraries like "System.Text.Json" taken from "Azure.Core". That's why I am not directly referring to "System.Text.Json" here.

Reply:
Do we use any other type from Azure.Core?

Reply:
I found this only so far. Another reason to use "Azure.Core" is that it could be hard to decide the proper version of "System.Text.Json" to depend on. I tried with the latest version of "System.Text.Json" but it gave me lots of build errors regarding version conflicts or package downgrades. So I just refer to how we depend on "Azure.Core" on the [service ](https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/iothub/service/src/Microsoft.Azure.Devices.csproj#L63)client.

Reply:
We get ETag from Azure.Core, IIRC. Also, the JsonPatchDocument, but that isn't in our API surface, just as a recommendation, I think.